frontend/src/wrapper/TopNavigation.tsx

```diff
@@ -97,14 +102,14 @@ export default function TopNavigation({ drawerOpen = false, pageTopStyling = {},
       open={drawerOpen}
       position='absolute'
       data-test='appBar'
-      sx={{
+      sx={(theme) => ({
         ...pageTopStyling,
         top: 'unset',
-        background:
-          theme.palette.mode === 'light'
-            ? 'linear-gradient(276deg, rgba(214,37,96,1) 0%, rgba(84,39,142,1) 100%)'
-            : '#242424',
-      }}
+        background: '#242424',
```

Comment:
Maybe add a todo to use `theme.applyStyles` to set this for dark mode only in a future PR?